packages/ws-server/src/webSocketServer.ts

```diff
@@ -104,7 +104,8 @@ app.ws.use((ctx) => {
 
             if (event === 'logs') {
                 if(!getMultipleAddressesEnabled() && Array.isArray(filters.address) && filters.address.length > 1) {
-                    response = predefined.INVALID_PARAMETER(4, 'Only one address is allowed');
+                    // [1] is the second object in the expected array of parameters that contains the filters (address and topics) for the subscription
+                    response = jsonResp(request.id, predefined.INVALID_PARAMETER("params[1].address", 'Only one contract address is allowed'), undefined);
```

Comment:
nit: use filters instead of `params[1]`
Q: Also do any of these params need validation before use?

Maybe a separate ticket/PR to ensure safety

Reply:
✅  Changed `params[1]` to `filters` completed 

Also, good catch on parameters sanity check, I have created this ticket to track parameter validations.
https://github.com/hashgraph/hedera-json-rpc-relay/issues/1045

Please add any other validations that you consider necessary besides the ones I have already included.